packages/app-store/hubspotothercalendar/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "private": true,
+  "name": "@calcom/hubspotothercalendar",
+  "version": "0.0.0",
+  "main": "./index.ts",
+  "description": "HubSpot is a cloud-based CRM designed to help align sales and marketing teams, foster sales enablement, boost ROI and optimize your inbound marketing strategy to generate more, qualified leads.",
+  "dependencies": {
+    "@calcom/lib": "*",
+    "@calcom/prisma": "*",
+    "@hubspot/api-client": "^6.0.1-beta4"
```

Comment:
I will keep an eye on the coming releases for the `@hubspot/api-client` package as ideally we shouldn't be using beta versions. I'm already watching the correspondent repo for new releases to be covered here.